app/_hub/tomkerkhove/microsoft_azure/index.md

```diff
@@ -0,0 +1,158 @@
+---
+name: Microsoft Azure
+publisher: Tom Kerkhove
+
+categories:
+  - deployment
+
+type: integration
+
+desc: Deploy Kong on Microsoft Azure # (required) 1-liner description; max 80 chars
+description: #|
+  This guide walks you through the options to run Kong on Microsoft Azure.
+  # (required) extended description.
+  # Use YAML piple notation for extended entries.
+  # EXAMPLE long text format (do not use this entry)
+  # description: |
+  #   Maintain an indentation of two (2) spaces after denoting a block with
+  #   YAML pipe notation.
+  #
+  #   Lorem Ipsum is simply dummy text of the printing and typesetting
+  #   industry. Lorem Ipsum has been the industry's standard dummy text ever
+  #   since the 1500s.
+
+support_url: # (Optional) A specific URL of your own for this extension.
+  # Defaults to the url setting in your publisher profile.
+
+source_url: https://github.com/tomkerkhove/kong-deployment-on-azure
+  # (Optional) If your extension is open source, provide a link to your code.
+
+#license_type:
+  # (Optional) For open source, use the abbreviations in parentheses at:
+  # https://opensource.org/licenses/alphabetical
+
+#license_url:
+  # (Optional) Link to your custom license.
+
+#privacy_policy:
+  # (Optional) If you have a custom privacy policy, place it here
+
+#privacy_policy_url:
+  # (Optional) Link to a remote privacy policy
+
+#terms_of_service:
+  # (Optional) Text describing your terms of service.
+
+#terms_of_service_url:
+  # (Optional) Link to your online TOS.
+
+# COMPATIBILITY
+# In the following sections, list Kong versions as array items.
+# Versions are categorized by Kong edition and their known compatibility.
+# Unlisted Kong versions will be considered to have "unknown" compatibility.
+# Uncomment at least one of 'community_edition' or 'enterprise_edition'.
+# Add array-formatted lists of versions under their appropriate subsection.
+
+kong_version_compatibility: # required
+  community_edition: # optional
+    compatible:
+        - 0.14.x
```

Comment:
Same comments here as the compatibility section in the other file - I think it needs adjusting.